packages/react/src/hooks/useGetURL.ts

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import { getUrl, GetUrlInput, GetUrlOutput } from 'aws-amplify/storage';
+
+import { isFunction } from '@aws-amplify/ui';
+import { useHasValueUpdated } from '@aws-amplify/ui-react-core';
+
+type UseGetUrlInput = GetUrlInput & {
```

Comment:
Prefer `interface` over `type` where possible

Reply:
I had tried that initially but get this ts error `An interface can only extend an object type or intersection of object types with statically known members.`